src/cluster_legacy.h

```diff
@@ -289,6 +310,24 @@ static_assert(offsetof(clusterMsg, data) == 2256, "unexpected field offset");
                                               primary is up. */
 #define CLUSTERMSG_FLAG0_EXT_DATA (1 << 2) /* Message contains extension data */
 
+typedef struct {
+    char sig[4];     /* Signature "RCmb" (Cluster message bus). */
+    uint32_t totlen; /* Total length of this message */
+    uint16_t ver;    /* Protocol version, currently set to CLUSTER_PROTO_VER. */
+    uint16_t notused1;
+    uint16_t type; /* Message type */
+    union clusterMsgDataLight data;
+} clusterMsgLight;
```

Comment:
Could we also add a `static_assert` for the total size of the struct?

Reply:
We discussed about this offline, the size is different for 32 bit and 64 bit architecture hence difficult to put it. So, avoiding it.